internal/store/device.go

```diff
@@ -659,19 +659,25 @@ func (s *DeviceStore) GetRendered(ctx context.Context, orgId uuid.UUID, name str
 		consoles = &[]api.DeviceConsole{*console}
 	}
 
-	renderedConfig := api.RenderedDeviceSpec{
-		RenderedVersion: renderedVersion,
-		Config:          device.RenderedConfig,
-		Os:              device.Spec.Data.Os,
-		Systemd:         device.Spec.Data.Systemd,
-		Resources:       device.Spec.Data.Resources,
+	device, err := deviceModel.ToApiResource()
+	if err != nil {
+		return nil, err
+	}
+
+	spec := api.DeviceSpec{
```

Comment:
Nit: In the future we may add more fields to device spec.  Why not just update config, applications, and consoles in device.Spec, so that all other fields go through as-is?

Reply:
moved this into the model